docs/usage/authentication.md

```diff
@@ -0,0 +1,44 @@
+There are several ways to authenticate using the SDK:
+
+```python
+from pasqal_cloud import SDK
+
+project_id = "your_project_id"  # Replace this value with your project_id on the PASQAL platform. It can be found on the user-portal, in the 'project' section.
+username = "your_username"  # Replace this value with your username or email on the PASQAL platform.
+password = "your_password"  # Replace this value with your password on the PASQAL platform.
```

Comment:
In the comment, let's recommend the user not to hard-code the password but instead use environment variables to avoid leaking their password by mistake when pushing their code

Reply:
I've added a warning block for that, introducing the "admonition" extension of the MkDocs-Material theme.